Zotero/Scenes/Detail/Items/Views/ItemsFilterViewController.swift

```diff
@@ -98,7 +98,7 @@ class ItemsFilterViewController: UIViewController {
         self.downloadsTitleLabel.text = L10n.Items.Filters.downloads
         self.downloadsSwitch.isOn = self.downloadsFilterEnabled
 
-        guard UIDevice.current.userInterfaceIdiom == .phone else {
+        guard traitCollection.horizontalSizeClass == .compact || UIDevice.current.userInterfaceIdiom == .phone else {
```

Comment:
Similar as above, doesn't this need an update on size class change?

Reply:
Check https://github.com/zotero/zotero-ios/pull/762/commits/882576cc235304112997182051826044dcc89adb